docs/examples/extensions/add-report/woocommerce-admin-add-report-example.php

```diff
@@ -39,8 +39,9 @@ function add_report_register_script() {
  */
 function add_report_add_report_menu_item( $report_pages ) {
 	$report_pages[] = array(
+		'id'     => 'example-analytics-report',
 		'title'  => __( 'Example', 'woocommerce-admin' ),
-		'parent' => '/analytics/revenue',
```

Comment:
@rrennick FYI, you'll want to make the same change once `0.13.0` in Scheduled Actions, otherwise the submenu item will disappear.

Reply:
Fixed in https://github.com/Prospress/action-scheduler-admin/pull/11.